README.md

```diff
@@ -64,6 +66,23 @@ Goals
   of the way for everything else).
 * No provisioning whatsoever; treat boxes as "fork and forget" for new projects.
 
+## Choosing between xAMP, Vagrant, and Docker
+
+Deciding what local development environment to choose for you and your team
+can be tricky. Here are three options, ordered in terms of complexity:
+
+1. Is your team entirely new to PHP and web development in general? Consider
+   using something like [MAMP](https://www.mamp.info) instead of Vagrant or
+   Docker.
```

Comment:
Should we mention Windows options? Is WAMP still a thing?

Reply:
MAMP actually has a windows variant now (!) https://www.mamp.info/en/downloads/

I don't know anything current to actually recommend for Windows though. Open to anything we know is good.

Reply:
> MAMP actually has a windows variant now (!)

😲 

Disregard then!